src/plugins/workspace/public/components/workspace_detail/workspace_detail_form_content.tsx

```diff
@@ -138,6 +146,38 @@ export const WorkspaceDetailFormContent = ({
           />
         </EuiCompressedFormRow>
       </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
```

Comment:
To show this section, shall we need to check if permission enabled?

Reply:
permission enabled should be checked, thanks!